main.tex

```diff
@@ -230,37 +235,37 @@
 
 \item  not interfere with any closed-circuit television monitoring devices, audio-visual cameras and other audio-visual surveillance devices; and
 
-\item  not to do or permit anything to be done that may prejudice the security or safety of the Common Property or any person in or around the Building.
+\item  not do (or permit to be done) anything that may harm the security or safety of the Common Property or any fellow Lot Owner or Occupier or their pets in or around the Building.
 
 \end{enumerate}
 
-\item  Without limiting the generality of the requirement of Rule 1.3(2)(a), an Owner or Occupier must not:
+\item  Without limiting the generality of the requirement of Rule 1.3(3)(a), an Owner or Occupier must not:
 
 \begin{enumerate}[label=(\alph*)]
 
-\item  allow any unknown guest(s) to enter the Building;
```

Comment:
Your friend brings their new boyfriend to your apartment, but you have to tell him that he's not allowed in the building, because you don't know him.